Src/AutoFixtureUnitTest/DomainNameGeneratorTest.cs

```diff
@@ -54,7 +54,7 @@ public void CreateReturnsOneOfTheFictiousDomains()
             var result = sut.Create(typeof(DomainName), null);
             // Verify outcome
             var actualDomainName = Assert.IsAssignableFrom<DomainName>(result);
-            Assert.True(Regex.IsMatch(actualDomainName.Domain, @"example\.(com|org|net)"));
+            Assert.Matches(@"example\.(com|org|net)", actualDomainName.Domain);
```

Comment:
Cudos to [xunit code analyzers](https://github.com/AutoFixture/AutoFixture/pull/773/commits/a451db59ca0708ea98cb346a4011307af701ed52) 👋